src/sentry/integrations/msteams/client.py

```diff
@@ -69,18 +73,27 @@ def update_card(self, conversation_id, activity_id, card):
 
 # MsTeamsPreInstallClient is used with the access token and service url as arguments to the constructor
 # It will not handle token refreshing
-class MsTeamsPreInstallClient(MsTeamsAbstractClient):
-    def __init__(self, access_token, service_url):
+class MsTeamsPreInstallClient(ApiClient, MsTeamsClientMixin):
+    integration_name = "msteams"
+
+    def __init__(self, access_token: str, service_url: str):
         super().__init__()
         self.access_token = access_token
         self.base_url = service_url.rstrip("/")
 
+    def request(self, method, path, data=None, params=None):
+        headers = {"Authorization": f"Bearer {self.access_token}"}
+        return self._request(method, path, headers=headers, data=data, params=params)
+
 
 # MsTeamsClient is used with an existing integration object and handles token refreshing
-class MsTeamsClient(MsTeamsAbstractClient):
-    def __init__(self, integration):
-        super().__init__()
+class MsTeamsClient(IntegrationProxyClient, MsTeamsClientMixin):
+    integration_name = "msteams"
```

Comment:
@scttcper It looks like we needed to set the `integration_name` property manually, since this is how Datadog tracks the metrics for each integration. Previously it was generating metrics of `sentry.integrations.http_response.undefined` instead of `sentry.integrations.http_response.msteams`.

I was able to fix this and added some tests to the relevant API clients to check for this.